canonical-websites/www.ubuntu.com
# 1763 Add submit visibility toggle to form on /legal/contributors/submit

## Done

- Add submit visibility toggle to form on /legal/contributors/submit
- Added spacing for consecutive fieldsets
- Linted existing JS file

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/legal/contributors/submit](http://0.0.0.0:8001/legal/contributors/submit))
- Toggle to radio buttons to verify the correct forms appear

## Issue / Card

Fixes #1763 

## Demo

[http://www.ubuntu.com-1763-submit-toggle.demo.haus/legal/contributors/submit](http://www.ubuntu.com-1763-submit-toggle.demo.haus/legal/contributors/submit)
